.pyup

```diff
@@ -0,0 +1,9 @@
+schedule: "every month"
+search: False
+requirements:
+  - requirements/default.txt:
```

Comment:
If I understand this correctly, the PRs that pyup will do won't be looking at constraints, so it's possible that we'll have to do follow-up commits to get things working. Does that sound right?

Reply:
That's what I intend. I'm not entirely sure whether we should or shouldn't be monitoring the constraints too, given how we're using it. I figure the security monitoring will include the constraints, at least.